configuration/device-tree.md

```diff
@@ -354,10 +407,29 @@ just_one    = <0>,"+1-2"; // Enable 1, disable 2
 conditional = <0>,"=3!4"; // Enable 3, disable 4 if value is true,
                           // otherwise disable 3, enable 4.
 ```
-The i2c-mux overlay uses this technique.
+The `i2c-rtc` overlay uses this technique.
 
-<a name="part2.2.5"></a>
-#### 2.2.5 Examples
+#### 2.2.9 Special properties
+<a name="part2.2.9"></a>
+
+A few property names, when targeted by a parameter, get special handling. One you may have noticed already - `status` - which will convert a boolean to either `okay` for true and `disabled` for false.
+
+Assigning to the `bootargs` property appends to it rather than overwriting it - this is how settings can be added to the kernel command line.
+
+The `reg` property is used to specify device addresses - the address a memory-mapped hardware block appears, the address on an I2C bus, etc. The names of child nodes should be qualified with their addresses in hexadecimal, using `@` as a separator:
```

Comment:
"the address a memory-mapped hardware block appears" doesn't read right.

Reply:
I'm not convinced it's wrong, but I'll change it to "the address where...".